controllers/numaresourcesscheduler_controller.go

```diff
@@ -194,3 +201,23 @@ func (r *NUMAResourcesSchedulerReconciler) updateStatus(ctx context.Context, sch
 	}
 	return nil
 }
+
+func (r *NUMAResourcesSchedulerReconciler) syncConfigMapHash(ctx context.Context) (string, error) {
```

Comment:
please add a reference as comment about an article/blog post which illustrates this technique